sql/fill-db.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO profiles (id, name, surname, age) VALUES(123456789, 'James', 'Bond', 40),
+(345678912, 'Albert', 'Einstein', 66),(444412365, 'Victor', 'Gugo', 22);
+SELECT * FROM profiles;
```

Comment:
Why do we need this select statement?
Remove it.

Reply:
Oops. It was for testing purposes only